### PR TITLE
FEAT: Added explore page

### PR DESCRIPTION
**Pull request by zoyatanweer**
Added explore page (without filter)
Files to check :
- components/Categories/Categories.jsx
- components/VideoCard/Videocard.jsx
- pages/Explore/Explore.jsx
- pages/Login/Login.jsx
- pages/Signup/Signup.jsx

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -36,3 +36,28 @@
...
+}
+.login-signup-form {
+  /* background-color: var(--form-bg-color); */
+  background-color: var(--explore-background);
+  border-radius: 10px;
+  width: 35%;
+  text-align: center;
+}
+.form-heading {
+  /* color: var(--navigation-color); */
```
Comment (dev-prabhat) [inline src/App.css:60]: remove commented code

File: src/App.js [hunk 2]
```diff
@@ -1,11 +1,27 @@
 import "./App.css";
+import { Routes, Route } from "react-router-dom";
+import { Header } from "./components/Header/Header";
 import { Homepage } from "./pages/Homepage/Homepage";
+import { Explore } from "./pages/Explore/Explore";
 import { Sidebar } from "./components/Sidebar/Sidebar";
+import { Footer } from "./components/Footer/Footer";
+import { Login } from "./pages/Login/Login";
+import { Signup } from "./pages/SignUp/SignUp";
```
Comment (dev-prabhat) [inline src/App.js:9]: export pages from a single file like an index.js

File: src/App.js [hunk 3]
```diff
@@ -1,11 +1,27 @@
...
   return (
     <div className="App">
-      <Homepage />
+      <Header />
+      <Routes>
+        <Route path="/" element={<Homepage />} />
+        <Route path="/explore" element={<Explore />} />
+        {/* <Route path="/playlist" element={<Playlist />} />
+        <Route path="/watchLater" element={<WatchLater />} /> */}
+        {/* <Route path="/explore" element={<Explore />} /> */}
```
Comment (dev-prabhat) [inline src/App.js:20]: remove commented code

**Review by dev-prabhat — CHANGES_REQUESTED** [review]
Suggested some changes

File: src/App.css [hunk 4]
```diff
@@ -23,10 +23,10 @@
   font-size: calc(10px + 2vmin);
   color: white;
 }
-
+/* 
 .App-link {
   color: #61dafb;
-}
+} */
```
Comment (Jia-A) [inline src/App.css:29]: This is commented code, right? You can remove it if not wanted!

File: src/Assets/Svg/allsvg.jsx [hunk 5]
```diff
@@ -172,6 +172,68 @@ function TwitterIcon(props) {
...
+      ></path>
+      <path
+        fill="currentColor"
+        d="M9 21h9c.83 0 1.54-.5 1.84-1.22l3.02-7.05c.09-.23.14-.47.14-.73v-2c0-1.1-.9-2-2-2h-6.31l.95-4.57l.03-.32c0-.41-.17-.79-.44-1.06L14.17 1L7.58 7.59C7.22 7.95 7 8.45 7 9v10c0 1.1.9 2 2 2zM9 9l4.34-4.34L12 10h9v2l-3 7H9V9zM1 9h4v12H1z"
+      ></path>
+    </svg>
+  );
+}
+
+// {/* <FontAwesomeIcon icon="fa-light fa-thumbs-up" />; */}
```
Comment (Jia-A) [inline src/Assets/Svg/allsvg.jsx:214]: Remove the commented code.

File: src/components/Categories/Categories.jsx [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+            <li
+              value={FilterState.categories}
+              onClick={(e) =>
+                FilterDispatch({
+                  type: "CATEGORIES",
+                  payload: e,
+                })
+              }
+              className="span-all"
+            >
```
Comment (Jia-A) [inline src/components/Categories/Categories.jsx:23]: Remove this commented code if not needed in the future.

File: src/components/Header/Header.css [hunk 7]
```diff
@@ -0,0 +1,10 @@
+.btn-primary {
+  background-color: var(--primary-color);
+  padding: 2px;
+  height: 40px;
```
Comment (Jia-A) [inline src/components/Header/Header.css:4]: Convert this to rem or em.

File: src/components/Header/Header.jsx [hunk 8]
```diff
@@ -1,16 +1,37 @@
...
+        </div>
+        <div className="user-profile">
+          {/* <UserIcon /> */}
+          <NavLink to="/login">
+            <button class="btn box-shadow btn-primary">Login</button>
+          </NavLink>
+        </div>
+      </nav>
+      <div className="division"></div>
+      {/* ----navigation ends---- */}
```
Comment (Jia-A) [inline src/components/Header/Header.jsx:32]: Good use of comment here Zoya!

File: src/pages/Explore/Explore.css [hunk 9]
```diff
@@ -0,0 +1,26 @@
...
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  margin-left: 2.4rem;
+}
+
+.filter-section > span {
+  /* background-color: var(--background-color); */
+  background-color: #343a40;
+  border-radius: 20px;
```
Comment (Jia-A) [inline src/pages/Explore/Explore.css:11]: change px to rem.

**Review by Jia-A — APPROVED** [review]
Code structure is good, you used comments to let the reader know what is happening which is a nice practice.
There is too much commented code which you might wanna remove so that your code will look clean and use something like Beautify in the end for proper indentation and space.
You can merge it if you want to work on the next part of the project and remove the unnecessary chunk later on.

**Review by dev-prabhat — APPROVED** [review]

**Comment by zoyatanweer** [comment]
thankyou for all of your suggestions, will surely do them.

**Pull request merged**